dependencies/url/validator.go

```diff
@@ -46,6 +46,7 @@ func init() {
 		"10.0.0.0/8",     // RFC1918
 		"172.16.0.0/12",  // RFC1918
 		"192.168.0.0/16", // RFC1918
+		"169.254.0.0/16", // RFC3927
 		"::1/128",        // IPv6 loopback
 		"fe80::/10",      // IPv6 link-local
```

Comment:
One more thing that you might want to note here, if there are other address spaces that need to be added here you might want to add them together next time. Appreciate this!